pkg/pool/linked_list_test.go

```diff
@@ -0,0 +1,365 @@
+package pool
+
+import (
+	"reflect"
+	"testing"
```

Comment:
You should use the [assertify](https://github.com/stretchr/testify) package here, and instead of having your check functions return a bool, just call `assert.<>` inside them. It looks cleaner and you can avoid doing deep equals (there's an `assert.DeepEqual` func for you to use)

Reply:
Resolved in pkg/core/gsettingsbackend